python/ml4ir/base/tests/test_lr_schedule.py

```diff
@@ -217,7 +218,7 @@ def test_cyclic_lr_in_training_pipeline(self):
         lr_list = my_callback_object.get_lr_list()
         lr_gold = [0.001,0.020800006,0.040599994,0.0604,0.080199994,0.1,0.080199994,0.0604,0.040599994,0.020800006,0.001,
                    0.010900003,0.020800006,0.030699994,0.040599994,0.050499998,0.040599994,0.030699994,0.020800006,
-                   0.010900003,0.001,0.0059499955,0.010900003,0.015849996,0.020800006,0.02575,0.020800006,0.015849996,
+                   0.010900003,0.001,0.0059499955,0.010900003,0.015849996,0.020800006,0.02575,0.02575,0.020800006,0.015849996,
```

Comment:
Thoughts on why this changed?

Reply:
This was surprising to me too. If you notice the values, it is just one single value that got changed. In the new version, there are 51 values compared to 50 previously. 

Here are a few guesses:
- There is a new implementation on tensorflow side in `tf.keras.optimizers.schedules.LearningRateSchedule`
- I do call `model(...)` once before calling `model.fit(...)`. Maybe the learning rate scheduler also gets triggered with this.

I haven't investigated further here though because the change looked very small to me. But might be good to hear from @mohazahran too as he wrote this piece originally.